package.json

```diff
@@ -9,14 +9,15 @@
         "url": "https://github.com/SamWSoftware/serverless-auto-swagger"
     },
     "scripts": {
-        "build": "tsc && cp ./src/resources/*.py ./dist/resources/",
+        "build": "tsc && copyfiles -f ./src/resources/*.py ./dist/resources/",
         "prepare-release": "standard-version",
         "prepublish": "npm run build",
         "precommit": "npm run test",
         "publish": "np",
         "test": "jest"
     },
     "dependencies": {
+        "copyfiles": "^2.4.1",
```

Comment:
`copyfiles` could be a dev dependency since it's used only for `npm run build`, not for the actual package itself, right?

Reply:
Done, thanks :)

As i'm not a node/js developer, i just installed it normally by npm install and npm put it there. I'll move it :)